packit_service/worker/helpers/build/copr_build.py

```diff
@@ -80,8 +80,8 @@ def __init__(
         metadata: EventData,
         db_project_event: ProjectEventModel,
         job_config: JobConfig,
-        build_targets_override: Optional[set[str]] = None,
-        tests_targets_override: Optional[set[str]] = None,
+        build_targets_override: Optional[set[tuple[str, str]]] = None,
+        tests_targets_override: Optional[set[tuple[str, str]]] = None,
```

Comment:
I have a feeling you're changing the meaning of this attribute, so the name no longer holds very well, but… I guess that's a different rabbit hole to go down…

Reply:
I would say they are still targets override but a bit more specific. They are targets for a specific identifier.

The most efficient structure here is probably a dictionary and not a `set` anymore, however I would start with this simpler enhancement.